vtexdocs/helpcenter
# Fix components' localization and improve navigation file format

#### What is the purpose of this pull request?

Fix the localization of components' strings.

> Related to `components` [PR 32](https://github.com/vtexdocs/components/pull/32).

#### How should this be manually tested?

1. Sync with this branch locally: `fix/components-localization-retry`.
2. Change the components import to the correct branch:
    "@vtexdocs/components": "https://github.com/vtexdocs/components.git#fix/localized-strings-retry",
3. Test different pages and check components' strings, such as the side bar title, the search box place holder and the `was this helpful` option. They should be localized in PT and ES.
